controller/helper/process-patient-status.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+/**
+ * @module controller/handler/process-patient-status
+ */
+
+/**
+ * A dashboard with an overview of a specific trial.
+ * @param {Array<Object>} rows - aggregated survey information
+ * @returns {Array<Object>} rows - updated rows object with compliance status set for each record
```

Comment:
line 7-9 update docs to reflect the new purpose and the correct incoming and outgoing types